src/website/views/components/breadcrumb.pug

```diff
@@ -1,4 +1,158 @@
 ---
 title: Breadcrumb
-disabled: true
 ---
+
+p.-text Breadcrumb indicate the current page’s location within a navigational hierarchy.
+
+h3 Examples
+p.-text
+  | To render a breadcrumb, use the class <code>a-breadcrumb</code>.
+  | A contextual class (e.g. <code>-separatorIcon</code>) is required to provide alternative separator styling.
+
+h4 Base
+.example.-mb3
+  .-p1
+    nav.a-breadcrumb.-text
```

Comment:
Let's add `aria-label="Breadcrumb"` to each `<nav>` element as defined [here](https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html) in w3 best practices. We'll also need `aria-current="page"` on the anchor of the active page, in this case, Child 2.